src/PJ_isea.c

```diff
@@ -59,13 +60,19 @@ int hex_iso(struct hex *h) {
 }
 
 ISEA_STATIC
-int hexbin2(double width, double x, double y,
-                int *i, int *j) {
+void hexbin2(double width, double x, double y, int *i, int *j) {
     double z, rx, ry, rz;
     double abs_dx, abs_dy, abs_dz;
     int ix, iy, iz, s;
     struct hex h;
 
+    if (pj_is_nan(x) || pj_is_nan(y) || pj_is_nan(width)) {
```

Comment:
The NaN-input can't be avoided earlier in the code? Without being particularly familiar with this code it seems like a fairly low-level function. What happens when `*i` and `*j` are set to 0? Does this still produce useful coordinates in the end?

Reply:
I will take another go at fixing this.  I'm sure it can be caught earlier.  

I always get nervous the higher up the stack I go.  I'd bet the results are garbage in the case of incoming nans.  Zeroing the result and complaining are better than the SIGILL I was getting.

Reply:
I can relate. But if it can be caught it is easier to fail gracefully I think. I wonder if this should really fail at the initialisation stage? The input doesn't seem to be unreasonable. Especially compared to the proj-strings :)